src/resolution_functions/models/panther_abins.py

```diff
@@ -14,24 +21,118 @@
 
 @dataclass(init=True, repr=True, frozen=True, slots=True, kw_only=True)
 class PantherAbINSModelData(ModelData):
+    """
+    Data for the `PantherAbINSModel` model.
+
+    Parameters
+    ----------
+    function
+        The name of the function, i.e. the alias for `PantherAbINSModel`.
+    citation
+        The citation for a particular model.
+    abs
+        Polynomial coefficients for the energy transfer (frequencies) polynomial, in order of
+        increasing degree, i.e. ``a, b, c`` for ``a + bx + cx^2``. Any number of coefficients is
+        allowed.
+    ei_dependence
+        Polynomial coefficients for the initial energy polynomial, in order of increasing degree,
+        i.e. ``a, b, c`` for ``a + bx + cx^2``. Any number of coefficients is allowed.
+    ei_energy_product
+        Polynomial coefficients for the product of initial energy and energy transfer (frequencies)
+        polynomial, in order of increasing degree, i.e. ``a, b, c`` for ``a + bx + cx^2``. Any
+        number of coefficients is allowed.
+
+    Attributes
+    ----------
+    function
```

Comment:
It seems redundant to explicitly document dataclass attributes that were set by init. I wonder what the general practices are around this.

Reply:
yeah, that's what I was wondering as well while copying massive amount of text; I was thinking of removing the `Parameters` section since the dataclasses should not be instantiated directly and only the attributes are potentially important to users, but idk.

Reply:
In fact they seem to be appearing three times because there is also some automatic documentation from the value definitions.

I think there is a way to suppress these. Alternatively, we could add docstrings to them and cut out most of the main docstring. (But then it is less unclear what is an `__init__` parameter.)

![Screenshot 2024-12-04 at 16 11 25](https://github.com/user-attachments/assets/86788b99-7a82-4854-921f-69e53df3bdb7)